code/api/src/modules/delivery/model.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
```

Comment:
To my understanding, this is the file is creating a new table 'Delivery' inside of our database?

Reply:
There's a folder under code/api/src/migrations that is actually how the new table's (or additions to existing tables) are created. From the command line we can create a migration file, open the file, edit it, and then enter another command that "runs" the migration. Once a migration is created AND run the additions are in the database locally and can be seeded with data. 
The model files(like this one) help us declare the data and data types of expected data in the tables of our database. Using sequelize.define allows us to be able to query the database directly with things like sequelize.delivery.status to find out the status of delivery. "Sequelize is a promise-based Node.js ORM for Postgres..." (and other types of databases).